scripts/validate-env.js

```
@@ -0,0 +1,123 @@
+  );
+
+  envVars.forEach(varName => {
+    if (!knownVars.includes(varName) && !varName.startsWith('VERCEL_')) {
+      warnings.push(`⚠️  Unknown environment variable: ${varName}`);
+    }
+  });
+
+  // Display results
+  if (warnings.length > 0) {
+    console.log('\n📝 Warnings:');
+    warnings.forEach(warning => console.log(warning));
```

Biome (2.4.6)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)